Tools/TestWebKitAPI/Tests/WTF/Vector.cpp

```diff
@@ -53,6 +58,23 @@ TEST(WTF_Vector, ZeroSize)
     EXPECT_EQ(nullptr, intVector.data());
     EXPECT_EQ(0U, intVector.size());
     EXPECT_EQ(0U, intVector.capacity());
+
+    auto intSpan = intVector.span();
+    EXPECT_EQ(nullptr, intSpan.data());
+    EXPECT_EQ(0U, intSpan.size());
+    EXPECT_EQ(0U, intSpan.size_bytes());
+
+    intVector.append(0);
+    intVector.removeLast();
+    EXPECT_TRUE(intVector.isEmpty());
+    EXPECT_NE(nullptr, intVector.data()); // data() is not nullptr.
```

Comment:
I don't think this comment is helpful.

Reply:
I added these comments not to document the code, but to call attention to the differences because it's easy to miss the "`NE`" instead of the "`EQ`" that the other checks have.